cypress/integration/finance/ledgers/ledgers.create.spec.js

```diff
@@ -0,0 +1,25 @@
+import Ledgers from '../../../support/fragments/finance/ledgers/Ledgers';
+
+import {
+  Button
+} from '../../../../interactors';
+
+describe('ui-finance: Ledger list search and filters', () => {
+  before(() => {
+    cy.login('diku_admin', 'admin')
```

Comment:
Could you use env variables please?
In master branch we have deafult changes with this values in cypress.json

`cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'))`